docs/tutorial/updating-our-app/index.md

```diff
@@ -89,9 +89,10 @@ much easier than having to look up the container ID and remove it.
 
 ### Starting our updated app container
 
-1. Now, start your updated app.
+1. Now, rebuild your image and start your updated app.
 
     ```bash
+    docker build -t getting-started
```

Comment:
The `docker build -t getting-started` step already is in line 22, before the trouble with the used port.

Reply:
I walked through the tutorial and I'm still happy with the current steps. Change the html file, then immediately build the image again and try to run it. The port conflict happens, then we show how to remove the old container and then filly run it again. But that happens all with the rebuilt image.

Is there anything we can improve to make this more visible?


![updating-docker-build](https://user-images.githubusercontent.com/207759/86048430-d0b69c80-ba50-11ea-8ccc-7f7ff26a56e9.jpg)

Reply:
I see that now - I followed the incorrect sequence of steps.  This means that #52 is closed.